CHANGELOG.md

```diff
@@ -0,0 +1,33 @@
+# Changelog
+
+* Friday 13th May 2022
+
+`Users needs are our strategy and our tactics` 
+v0.6.0
+## Features
+* We added new endFormat (for making sure your audio is the correct format for a target audience) this week we added Alexa preset 
+`code m = apiaudio.Mastering().create(scriptId=GLOBAL_SCRIPT_ID, endFormat="mp3_alexa")`
+* As a developer user I want to share my audio with my team/bosses/business person so they can try and test the wonders of api.audio. We call this *virality link* and it's available in the console now. 
+
+## Enterprise features
+* For our enterprise users we've enabled `sandboxing` this allows you to test safely API requests without using up credits. Please contact your account manager for further details. 
```

Comment:
Well, it's production without billing.

Reply:
Yeah but we'll add more stuff to it as well.